deployments/deployment.yaml

```diff
@@ -323,74 +323,16 @@ objects:
               - mountPath: /tmp
                 name: tmpdir
       jobs:
-        - name: retry-failed-tasks
+        - name: rename-domains
```

Comment:
this and line 335 might have been missed in the refactor. i think it should be `retry_failed_tasks`?

Reply:
good catch!

Reply:
fixed!